environment_testing.yml

```diff
@@ -0,0 +1,8 @@
+name: neo-test-env
+channels:
+  - conda-forge
+dependencies:
+  - datalad
+  - pip
+  - pip:
+    - -r requirements_testing.txt
```

Comment:
Why no remove this "requirements_testing.txt" and have everything at the same place ?

Reply:
To be nice to people who prefer to use pip for installing (almost) everything. You can still feed the `requirements_testing.txt` to pip if you prefer that, if I put everything in the yml this approach does not work any more..